app/javascript/components/pages/LoggedInHome.js

```diff
@@ -1,10 +1,55 @@
 import React, { Component } from 'react';
+import {
+  NavLink,
+  FormGroup,
+  Form,
+  Label,
+  Input,
+  Button,
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+
 
 class LoggedInHome extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      searchID: 0,
+      setPartyID: 0,
```

Comment:
```suggestion
      searchID: null,
      setPartyID: null,
```
initialize values should generally represent the data type or presence of data